src/styles/components/_hamburger.scss

```diff
@@ -0,0 +1,113 @@
+.hamburger {
+  position: absolute;
+  left: rem(16px);
+  width: 100%;
+
+  @include respond-above('md') {
+    display: none;
+  }
+
+  &__container {
+    user-select: none;
+
+    > span {
+      position: relative;
+      display: block;
+      width: rem(24px);
+      height: rem(3px);
+      border-radius: rem(3px);
+      margin-bottom: rem(5px);
+
+      background: $dark;
+
+      transform-origin: rem(4px 0px);
+
+      @include transition(transform, 0.55s, ease);
+      @include transition(cubic-bezier(0.77, 0.2, 0.05, 1) opacity, 0.5s, ease);
+    }
+  }
+
+  & span:nth-child(3) {
+    width: rem(16px);
+  }
+
+  & span:nth-child(4) {
+    margin: 0;
+    transform-origin: 0% 100%;
+  }
+
+  &__input {
+    position: absolute;
+    z-index: $zindex-hamburger-input;
+
+    display: block;
+    width: rem(24px);
+    height: 100%;
+    margin: 0;
+    opacity: 0;
+
+    cursor: pointer;
+
+    -webkit-touch-callout: none;
+
+    &:checked {
+      width: rem(20px);
+      height: 100%;
+    }
+
+    &:checked ~ .hamburger__menu {
+      opacity: 1;
+    }
+
+    &:checked ~ span {
+      opacity: 1;
+      transform: rotate(45deg) translate(0, -1px);
+    }
+
+    &:checked ~ span:nth-child(3) {
+      opacity: 0;
+      transform: rotate(0deg) scale(0.2, 0.2);
+    }
+
+    &:checked ~ span:nth-child(4) {
+      transform: rotate(-45deg) translate(3.5px, -1px);
+    }
+  }
+
+  &__menu {
+    position: absolute;
+
+    display: block;
```

Comment:
Since `display: block` is the default value of `ul` element, do we really need to override this property?

<img width="230" alt="Screen Shot 2564-04-07 at 10 16 16" src="https://user-images.githubusercontent.com/32285706/113805208-50b12600-978a-11eb-9b2b-6d3d5dd2c075.png">

Reply:
No need.